Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -103,7 +103,7 @@ public static class WebApiServiceCollectionExtensions
         /// <param name="services">Service collection to which to add authentication</param>
         /// <param name="configuration">Configuration</param>
         /// <returns></returns>
-        public static IServiceCollection AddProtectedApiCallsWebApis(
```

Comment:
>AddProtectedApiCallsWebApis [](start = 41, length = 27)

Since we do not use “protectedwebapp” so can we skip using “protectedwebapi” and just use WebApi?